jsonifier/models/models.py

```diff
@@ -105,6 +105,13 @@ def _jsonify_record(self, parser, rec, root):
                     value, json_key = self._jsonify_record_handle_resolver(
                         rec, field, resolver, json_key
                     )
+            # whatever json value we have found in subparser or not ass a sister key
+            # on the same level {jsoni_key}_fieldname
+            if rec.env.context.get("with_fieldname"):
+                json_key_fieldname = "_".join(["_fieldname", json_key])
```

Comment:
To me this is less readable than something like
`"_fieldname_" + json_key` or `f"_fieldname_{json_key}"`

Reply:
I've went  with a more readable version